client/components/helpCentre/HelpCentrePage.tsx

```diff
@@ -67,6 +67,11 @@ const HelpCentreRouter = () => {
 	*/
 
 	const ransomwareMessage = [
```

Comment:
Super minor thing, but should this be called something else as it has _multiple_ issues in it now? In fact, does it even need to be a separate var as it's being assigned to `knownIssues` immediately below?